signalfx/resource_signalfx_list_chart.go

```diff
@@ -251,6 +255,10 @@ func getListChartOptions(d *schema.ResourceData) *chart.Options {
 			if colorScaleOptions := getColorScaleOptions(d); len(colorScaleOptions) > 0 {
 				options.ColorScale2 = colorScaleOptions
 			}
+		} else {
+			if val, ok := d.GetOk("color_scale"); ok && val != nil {
+				return nil, fmt.Errorf("Using `color_scale` without `color_by = \"Range\"` has no effect")
```

Comment:
This error condition is confusing.  This `else` block is hit when `color_by != "Scale"` so potentially couldn't `color_by == "Range"` here?

Reply:
Oops!